src/hardware/rdtech-dps/protocol.c

```diff
@@ -259,13 +277,13 @@ SR_PRIV int rdtech_dps_get_state(const struct sr_dev_inst *sdi,
 	struct dev_context *devc;
 	struct sr_modbus_dev_inst *modbus;
 	gboolean get_config, get_init_state, get_curr_meas;
-	uint16_t registers[12];
+	uint16_t registers[14];
 	int ret;
 	const uint8_t *rdptr;
 	uint16_t uset_raw, iset_raw, uout_raw, iout_raw, power_raw;
 	uint16_t reg_val, reg_state, out_state, ovpset_raw, ocpset_raw;
 	gboolean is_lock, is_out_enabled, is_reg_cc;
-	gboolean uses_ovp, uses_ocp;
+	gboolean uses_ovp, uses_ocp, uses_12V_range = FALSE;
```

Comment:
See above: variable declaration / assignment

Reply:
Fixed in commit b7672857